### PR TITLE
feat: Cube search does not use full text search engine

### PR DESCRIPTION
**Pull request by ptbrowne**
Removing full text search and use simple contain for search.﻿

### CODE REVIEW

**Comment by ptbrowne** [comment]
@adintegra I tried the approach of not using the full text search here : https://github.com/visualize-admin/visualization-tool/issues/683. I think it solves the problem of performance. Since I had no longer access to scores, I had to redo the scoring logic but it's good, I think we could even reintroduce the keyword highlighting by doing it ourselves.

@ortnever and @rdataflow Could you try some of your favorite searches on https://visualization-tool-git-feat-search-no-fulltext-ixt1.vercel.app/en/browse?dataSource=Prod to see if it matches your expectations ? Thanks

**Review by bprusinowski — APPROVED** [review]
LGTM! 👌

File: app/rdf/query-search.spec.ts [hunk 1]
```diff
@@ -13,29 +13,18 @@ jest.mock("@tpluscode/sparql-builder", () => ({}));
...
-
-  it("should compute weighted score per cube from score rows, , and keep cube with score: 0", () => {
-    const reduced = computeScores(scores, {
-      keepZeros: false,
-    });
-    expect(reduced["a"].score).toEqual(7);
-    expect(reduced["b"].score).toEqual(25);
+    expect(reduced["a"].score).toEqual(5);
+    expect(reduced["b"].score).toEqual(7);
     expect(reduced["c"].score).toEqual(1);
```
[the remark this answers is not in the document]
Comment (ptbrowne) [reply app/rdf/query-search.spec.ts]: Thanks, done !

**Comment by ortnever** [comment]
@ortnever and @Rdataflow Could you try some of your favorite searches on https://visualization-tool-git-feat-search-no-fulltext-ixt1.vercel.app/en/browse?dataSource=Prod to see if it matches your expectations ? Thanks

It looks very good. Even with a short abstract from the description, the result is displayed in 1-3 seconds. Thank you !

**Pull request merged**